### ISSUE TITLE
Add player's current team to database entries

### ISSUE THREAD

**Issue by riFaulkner** [riFaulkner/sports-kernel#56]
Currently no players have a team under the attributes 'teamNFL' update that so player teams can be displayed.

**Comment by riFaulkner** [comment]
Awesome, as I'm working though creating the player cards to show players I'm starting to notice that there is a fair bit of info that would be really nice to be able to show. Their team, age and an avatar would really add a great level of polish to the FE and I think are reasonable lifts to manage.

**Comment by riFaulkner** [comment]
For team specifically, we should consider how we want to keep this data up-to-date. Player trades and Drops will be the most common things to happen to players, outside of playing games. We don't need a perfect solution now, for instance maybe we create a job that runs once a month (week?) and only updates the changes.

**Comment by KBThree13** [comment]
Agree. I've been thinking about this too. I think a couple pinch points for me are, what is our source for this? and where do we want to handle jobs like this? A low frequency data pull definitely gives us some options - though, I mostly wanted to avoid web scraping (as most websites' terms explicitly prohibit it).
